time_manager/time_manager.F90

```diff
@@ -1193,8 +1193,8 @@ function time_type_to_real(time)
 
 if(.not.module_is_initialized) call time_manager_init
 
-time_type_to_real = dble(time%days) * 86400.d0 + dble(time%seconds) + &
-     dble(time%ticks)/dble(ticks_per_second)
+time_type_to_real = real(dble(time%days) * 86400.d0 + dble(time%seconds) + &
+     dble(time%ticks)/dble(ticks_per_second), kind=r8_kind)
```

Comment:
Did this create a warning?  Everything is a double?

Reply:
double doesn't necessarily mean r8_kind so this had a warning but i think you'd only get it with gnu